pkg/ipam/allocator/azure/azure.go

```diff
@@ -48,6 +48,11 @@ func (*AllocatorAzure) Start(getterUpdater ipam.CiliumNodeGetterUpdater) (alloca
 
 	log.Info("Starting Azure IP allocator...")
 
+	azureCloudName, err := azureAPI.GetAzureCloudName(context.TODO())
```

Comment:
I can see there is at least one other instance of `context.TODO()`, do we have a plan to address these todos?

Reply:
@errordeveloper -- I changed the signature of the `Allocator` interface to pass a context to the `Start`.